include/seqan3/argument_parser/validators.hpp

```diff
@@ -275,7 +292,12 @@ class file_ext_validator
     void operator()(std::filesystem::path const & path) const
     {
         std::string ext{path.extension().string()};
-        ext = ext.substr(std::min(1, static_cast<int>(ext.size()))); // drop '.' if extension is non-empty
+        ext = ext.substr(std::min(1, std::max(0, static_cast<int>(ext.size())-1))); // drop '.' if ext is non-empty
+
+        // extensions were transformed to lower case during construction,so do the same for input path
+        if ( !case_sensitive )
```

Comment:
```suggestion
        if (!case_sensitive)
```
:nail_care: sorry :speak_no_evil:

Reply:
> :nail_care: sorry :speak_no_evil:

:unamused: